parachains/runtimes/collectives/collectives-polkadot/src/xcm_config.rs

```diff
@@ -175,7 +178,8 @@ impl Contains<RuntimeCall> for SafeCallFilter {
 				pallet_collective::Call::close_old_weight { .. } |
 				pallet_collective::Call::disapprove_proposal { .. } |
 				pallet_collective::Call::close { .. },
-			) => true,
+			) |
+			RuntimeCall::PolkadotXcm(pallet_xcm::Call::force_xcm_version { .. }) => true,
```

Comment:
@muharem @joepetrowski 
other cumulus runtimes do not need this `force_xcm_version` ?
what is the use-case with `force_xcm_version` for collectives?

Reply:
for bridge-hubs, I am testing if governance can call some dedicated calls with transact:
https://github.com/paritytech/cumulus/pull/2111/files#diff-70774adbfac7198ac09980fc1be181b5a22d99282ae82e454540dfd28e35febdR228-R267

Reply:
we set the version manually here https://github.com/paritytech/cumulus/pull/2221/files#diff-883b2a05c895554621314a4511df8dde5546f1d2ce910ac6ba5899a56c95bae1R85
otherwise xcm does it by itself, after the first xcm call, and if you test using parachains-integration-tests, it might mess the events and you get flaky tests.